src/summary/history/allergy-card-level-two.component.tsx

```diff
@@ -26,7 +26,7 @@ export function AllergyCardLevelTwo(props: AllergyCardLevelTwoProps) {
 
       return () => abortController.abort();
     }
-  }, [patient.identifier[0].value]);
+  }, [isLoadingPatient, patient]);
```

Comment:
same comment here, why have you added `isLoadingPatient`?

Reply:
It is a dependency to React useEffect hook